### PR TITLE
20-create About-us page

### LINKED ISSUE #34: 20 - Create about-us page
[About us](https://www.figma.com/file/MpfNES0KuBUkkd0Qk4EyOc/HYF-Connect-Wireframe?node-id=18%3A4)

### PR DESCRIPTION
**Pull request by firity**

### CODE REVIEW

**Review by dirk-jacobs — CHANGES_REQUESTED** [review]
Good work !
Some smaller changes:
- Fix the links to the members (/pages/members/...), site (https://hackyourfuture.be/) etc.
- Some small errors in the team-members circle pictures
   - Maria (no picture)
   - Inggritt ( oval )
   - Reinaldo ( slightly oval )

**Comment by firity** [comment]
@dirk-jacobs thank you for the review

- [x] Fix the links to the members (/pages/members/...), site (https://hackyourfuture.be/) etc.
- [x] Maria (no picture)
- [x] Inggritt ( oval )
- [ ]  Reinaldo ( slightly oval )

File: client/pages/about-us/main-section.js [hunk 1]
```diff
@@ -1,9 +1,147 @@
...
-    <div class="main">
-      <h1>{{message}}</h1>
-    </div>
+template: `
+
+<div class="about-us-container">
+<h1>About us</h1>
+</div>
+<div class="about-us">
+    <div class = "About-hyf">
```
Comment (dirk-jacobs) [inline client/pages/about-us/main-section.js]: PLease change all classes to lower-case

File: client/pages/about-us/main-section.js [hunk 2]
```diff
@@ -1,9 +1,147 @@
...
+</div>
+</div>
+
+<div class="About-title">
+<h4>people behind this project</h4>
+</div>
+<div class="About-main">
+<div class="profile-card">
+
+<div class="img">
```
Comment (dirk-jacobs) [inline client/pages/about-us/main-section.js]: class: "img" is not specific enough. Please prefix with about-

File: client/pages/about-us/profile-Card.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import ProfileComponent from "./profileCard-component.js";
```
Comment (dirk-jacobs) [inline client/pages/about-us/profile-Card.js]: please, call this file profilecard-component.js

Fix path to ProfileComponent
`import ProfileComponent from "./profile-component.js";`

File: client/pages/about-us/profile-Card.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    <template v-for="profile in profiles">
+      <profile-component:profile="profile"/>
+    </template>
+  </div>
+  </div>
+  `,
+  data() {
+    return {
+      Profiles: [
+        { img: "/images/members/MoaminA.png", caption: "Moamine","Full stack developer"},
```
Comment (dirk-jacobs) [inline client/pages/about-us/profile-Card.js]: There is an error in the json.
It should be something like:

`{ img: "/images/members/MoaminA.png", title: "Moamine", subtitle:"Full stack developer"},`

File: client/pages/about-us/profile-component.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+const ProfileComponent = {
+    props: ['profile'],
+    template: `  
+    <div class="About-main">
+        <div class="profile-card--img"><img v-bind:src="profile.img"></div>
+        <div class="profile__card--caption">{{profile.caption}}</div>
```
Comment (dirk-jacobs) [inline client/pages/about-us/profile-component.js]: I would make it something like
 ```
        <div class="profile-card--img"><img v-bind:src="profile.img"></div>
        <div class="profile__card--title">{{profile.title}}</div>   
        <div class="profile__card--subtitle">{{profile.subtitle}}</div>     

```

File: client/pages/about-us/profile-Card.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+const ProfileCard = {
+  components: {
+    ProfileComponent,
+  },
+  template: `
+  <div class="About-main">
+  <div class="profile-card">
+    <template v-for="profile in profiles">
+      <profile-component:profile="profile"/>
```
Comment (dirk-jacobs) [inline client/pages/about-us/profile-Card.js]: There is a missing space

`<profile-component :profile="profile"/>`

File: client/styles/styles.css [hunk 7]
```diff
@@ -472,3 +472,202 @@ input[type="password"] {
...
+.profile-card .img{
+  position: relative;
+  width: 100%;
+  height: 100%;
+  transition: .6s;
+  z-index: 99;
+
+}
+
+.img img{
```
Comment (dirk-jacobs) [inline client/styles/styles.css]: Please add about- prefix to the class name something like about-img

And... as a quick fix for the 'ovals'


```
.img img{
  width: 100%;
  min-width: 20%;
  border-radius: 50%;
  box-shadow: 0 0 22px #3336;
  transition: .6s;  
}
```

**Review by dirk-jacobs — CHANGES_REQUESTED** [review]
I suggested a number of changes. A quick-fix could have a valid page.
I would like to continue with the components.
What you did so far is not bad ! You're close

Start by doing the changes and also add the classes to the style.css which are in the components

File: client/styles/styles.css (hunk 7 above)
Comment (firity) [reply client/styles/styles.css]: ok, I will fix them

File: client/pages/about-us/main-section.js [hunk 8]
```diff
@@ -1,9 +1,147 @@
...
+<div class="about-us">
+    <div class = "about-hyf"> 
+        <p><strong>HackYourFuture Belgium </strong>is an initiative of Open Knowledge Belgium.
+    It is a coding school to support those who have limited access to education and labor market
+    Operated in Brussels and Antwerp. <br> <br>
+    <strong>HackYourFuture aims</strong> to enable 
+    refugees and disadvantaged people to build digital skills for a career in web development, 
+    facilitate the integration of newcomers, and address the shortage of qualified workforce in the IT sector.</p>
+    <br>
+    <a class="about-visit-website" href="https://hackyourfuture.be/">visit website</a>
```
Comment (Moamin-AbuEwaida) [inline client/pages/about-us/main-section.js]: can you add ( target="_blank" ) after the class so when the user clicks on the button, will open HYF website in another tab/window

File: client/pages/about-us/profile-Card.js [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+    <template v-for="profile in profiles">
+      <profile-component :profile="profile"/>
+    </template>
+  </div>
+  </div>
+  `,
+  data() {
+    return {
+      Profiles: [
+        { img: "/images/members/MoaminA.png", title: "Moamine", subtitle:"Full stack developer"},
```
Comment (Moamin-AbuEwaida) [inline client/pages/about-us/profile-Card.js]: my name is Moamin :stuck_out_tongue:

**Review by Moamin-AbuEwaida — COMMENTED** [review]
I shared few minor comments here as well as in slack

**Comment by rayanejsilva** [comment]
@firity I think the page is almost done. I would change these topics before merging. 

1. Reinaldo and Moamin images are slightly bigger. Because of that they look more oval. Can you resize the pictures?
2. I would delete the second Moamin's profile card and replace with Axil's picture.
3. Your name and Moamin names are incorrect.
4. It would be nice to add target attributes to the buttons.
5. You forgot to delete `a` tag in main-section.js. 
6.  Line 26 in main-section.js. it is HYF not HYC.
7. Pay attention to file naming. We don't use uppercase and we separate words with - . (E.g.  profile-card.js and profile-card-component.js

File: client/styles/styles.css (hunk 7 above)
Comment (dirk-jacobs) [reply client/styles/styles.css]: I made a mistake.
It needs to be 

```
.about-img img{
  width: 160px;
  height: 160px;
  object-fit: cover;
  border-radius: 50%;
  box-shadow: 0 0 22px #3336;
  transition: .6s; 
}
```

File: client/styles/styles.css [hunk 10]
```diff
@@ -472,3 +472,200 @@ input[type="password"] {
...
+.profile-card .about-img{
+  position: relative;
+  width: 100%;
+  height: 100%;
+  transition: .6s;
+  z-index: 99;
+
+}
+
+.about-img img{
```
Comment (dirk-jacobs) [inline client/styles/styles.css:720]: I made a mistake.
It needs to be 

```
.about-img img{
  width: 160px;
  height: 160px;
  object-fit: cover;
  border-radius: 50%;
  box-shadow: 0 0 22px #3336;
  transition: .6s; 
}

File: client/pages/about-us/profile-Card.js [hunk 11]
```diff
@@ -0,0 +1,34 @@
+import ProfileCardComponent from "./profileCard-component.js";
```
Comment (dirk-jacobs) [inline client/pages/about-us/profile-Card.js]: Make all **filenames lowercase** please

File: client/pages/about-us/profile-Card.js [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+
+const ProfileCard = {
+  components: {
+    ProfileCardComponent,
+  },
+  template: `
+  <div class="about-main">
+  <div class="profile-card">
```
Comment (dirk-jacobs) [inline client/pages/about-us/profile-Card.js]: Remove this div, not needed

File: client/pages/about-us/profile-Card.js [hunk 13]
```diff
@@ -0,0 +1,34 @@
...
+    <template v-for="profile in profiles">
+      <profile-component :profile="profile"/>
```
Comment (dirk-jacobs) [inline client/pages/about-us/profile-Card.js]: Should be

`<profileCard-component :profile="profile"/>`

File: client/pages/about-us/profile-Card.js [hunk 14]
```diff
@@ -0,0 +1,34 @@
...
+    </template>
+  </div>
+  </div>
+  `,
+  data() {
+    return {
+      Profiles: [
```
Comment (dirk-jacobs) [inline client/pages/about-us/profile-Card.js]: lowercase p

`profiles `

File: client/pages/about-us/profilecard-component.js [hunk 15]
```diff
@@ -0,0 +1,11 @@
+const ProfileCardComponent = {
+    props: ['profile'],
+    template: `
```
Comment (dirk-jacobs) [inline client/pages/about-us/profilecard-component.js:3]: If you want to make it exactly as what you have now, make it simple,
```
    <div class="profile-card">
        <div class="about-img">
            <img v-bind:src="profile.img">
        </div>
        <div class="caption">
        <h3>{{profile.title}}</h3>
        <p>{{profile.subtitle}}</p>
    </div>
```

**Review by dirk-jacobs — CHANGES_REQUESTED** [review]
A couple of changes + some xtras to work with components

If you want to use the component version in main-secion.js
You need to add 3 things


import ProfileCard from "./profile-Card.js";

    components: {
        ProfileCard,
      },

<profile-card></profile-card>

Then you can delete all the others.....

Enjoy

**Review by Moamin-AbuEwaida — APPROVED** [review]
LGTM

**Review by dirk-jacobs — APPROVED** [review]
A nice component based implementation.
Let's merge this

**Pull request merged**